src/app/components/Timestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -24,7 +94,6 @@ exports[`Timestamp should render Timestamp with datetime attribute and text 1`]
 
 <time
   className="c0"
-  dateTime="1530947227000"
```

Comment:
Just noticed this has been removed -- I think when using the `as` polymorphic prop?

Reply:
Fixed -- doesn't get passed through unless snake-cased.